recipes/r-report/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set version = '0.5.8' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-report
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/report_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/report/report_{{ version }}.tar.gz
+  sha256: 017320c21373cc9de8061951dd95147f2468fabc8a9d12fcc516a50c65c4383b
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-bayestestr >=0.13.1
+    - r-datawizard >=0.9.0
+    - r-effectsize >=0.8.6
+    - r-insight >=0.19.7
+    - r-parameters >=0.21.3
+    - r-performance >=0.10.8
+  run:
+    - r-base
+    - r-bayestestr >=0.13.1
+    - r-datawizard >=0.9.0
+    - r-effectsize >=0.8.6
+    - r-insight >=0.19.7
+    - r-parameters >=0.21.3
+    - r-performance >=0.10.8
+
+test:
+  commands:
+    - $R -e "library('report')"           # [not win]
+    - "\"%R%\" -e \"library('report')\""  # [win]
+
+about:
+  home: https://easystats.github.io/report/
```

Let's include the dev_url:

```suggestion
  home: https://easystats.github.io/report/
  dev_url: https://github.com/easystats/report/
```